ui/i18n/src/main/res/values/strings.xml

```diff
@@ -693,7 +693,7 @@
     <string name="authentication_descr">Change your username and password for this podcast and its episodes.</string>
     <string name="feed_tags_label">Tags</string>
     <string name="feed_tags_summary">Change the tags of this podcast to help organize your subscriptions</string>
-    <string name="feed_folders_include_root">Show in main list</string>
+    <string name="feed_folders_include_root">Show this subscription in main list</string>
```

Comment:
```suggestion
    <string name="feed_folders_include_root">Show podcast in main list</string>
```

Reply:
I'm trying to deliberately avoid the word "podcast" as much as possible because it is ambiguous. Especially here when some users already request putting episodes into folders.